Packs/CofenseTriage/Layouts/layoutscontainer-Cofense_Triage_Report.json

```diff
@@ -0,0 +1,686 @@
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "cofensetriagereportprocessedat",
+                                "height": 22,
+                                "id": "c1a0eb90-c43d-11eb-abed-c9810bb6a8e6",
+                                "index": 4,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "cofensetriagereportupdatedat",
+                                "height": 22,
+                                "id": "c5447820-c43d-11eb-abed-c9810bb6a8e6",
+                                "index": 5,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxH": null,
+                        "maxW": 1,
+                        "minH": 1,
+                        "minW": 1,
+                        "moved": false,
+                        "name": "Date \u0026 Time Information",
+                        "static": false,
+                        "w": 1,
+                        "x": 0,
+                        "y": 2
+                    },
+                    {
+                        "displayType": "ROW",
+                        "h": 2,
+                        "hideName": false,
+                        "i": "rskhiiumyx-d29c9b60-c43d-11eb-abed-c9810bb6a8e6",
+                        "items": [
+                            {
+                                "endCol": 2,
+                                "fieldId": "cofensetriagereportmd5",
+                                "height": 22,
+                                "id": "14edb030-c43e-11eb-abed-c9810bb6a8e6",
+                                "index": 0,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            },
+                            {
+                                "endCol": 2,
+                                "fieldId": "cofensetriagereportsha256",
+                                "height": 22,
+                                "id": "19867230-c43e-11eb-abed-c9810bb6a8e6",
+                                "index": 1,
+                                "sectionItemType": "field",
+                                "startCol": 0
+                            }
+                        ],
+                        "maxH": null,
+                        "maxW": 1,
+                        "minH": 1,
+                        "minW": 1,
+                        "moved": false,
+                        "name": "Hash Signatures",
```

Comment:
If possible please display in the layout the hashes of the attachment and not the hashes of the reports.

Reply:
sure @evisochek , we're now showing the hashes of the attachment